File: Golang/abc199_a/main.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	scanner := bufio.NewScanner(os.Stdin)
	scanner.Scan()
	line := scanner.Text()
	fields := strings.Fields(line)

	a, _ := strconv.Atoi(fields[0])
	b, _ := strconv.Atoi(fields[1])
	c, _ := strconv.Atoi(fields[2])

	result := solve(a, b, c)

	fmt.Println(result)
}

func solve(a int, b int, c int) string {
	if a*a+b*b < c*c {
		return "Yes"
	}

	return "No"
}

File: R/abc199_a/main.R
solve <- function(a, b, c) {
  if (a^2 + b^2 < c^2) {
    return("Yes")
  } else {
    return("No")
  }
}

main <- function() {
  line <- readLines("stdin", n = 1)
  nums <- as.integer(strsplit(line, " ")[[1]])
  result <- solve(nums[1], nums[2], nums[3])
  cat(result, "\n", sep="")
}

main()
